src/View/BookingCodes.php

```diff
@@ -59,10 +353,13 @@ public static function renderCSV( $timeframeId = null ) {
 		if ( $timeframeId == null ) {
 			$timeframeId = intval( $_GET['post'] );
 		}
-		$bookingCodes = \CommonsBooking\Repository\BookingCodes::getCodes( $timeframeId );
+
+        $timeframe=new Timeframe($timeframeId);
+		$bookingCodes = \CommonsBooking\Repository\BookingCodes::getCodes( $timeframeId, Wordpress::getUTCDateTime("today")->getTimestamp(), $timeframe->getRawEndDate() );
+
 		header( 'Content-Encoding: UTF-8' );
 		header( 'Content-type: text/csv; charset=UTF-8' );
-		header( "Content-Disposition: attachment; filename=buchungscode-" . commonsbooking_sanitizeHTML( $timeframeId ) . ".txt" );
+		header( "Content-Disposition: attachment; filename=bookingcodes-" . commonsbooking_sanitizeHTML( $timeframeId ) . ".txt" );
```

Comment:
Soll das hier ne txt bleiben oder soll das ne CSV auch werden?

Reply:
Schauen wir da auf den gleichen Stand des Codes? Ich habe die nextCronEmailEvent längst rausgenommen (https://github.com/wielebenwir/commonsbooking/pull/1174#discussion_r1161118899)?

Ansonsten zu dem type des Buchungscode downloads:
Ich habe das ja so vom master übernommen und gehe immer davon aus, dass Menschen sich etwas dabei gedacht haben. Wenn das nicht (mehr) der Fall ist wäre ich absolut dafür die fileextension auf .csv zu ändern, da ja auch der Content-type entsprechend ist. ABER: dann würde ich auch vorschlagen, beim erzeugen der Records statt '\t' den Field Separator ';' zu verwenden dann lässt es sich z.B. in Excel/Libre Office Calc ohne Änderungen direkt öffnen? 
Außer Menschen hatten vor die Textdatei direkt zu drucken dann ist natürlich der Tab besser, aber dann macht eigentlich dern Content-type: text/csv und Content-Transfer-Encoding: binary wenig Sinn.

Reply:
Wo hast du das genau gefunden mit der file extension?

Reply:
> Wo hast du das genau gefunden mit der file extension?

Hier:
https://github.com/wielebenwir/commonsbooking/blob/master/src/View/BookingCodes.php

Reply:
Ich denke auch wenn wir davon ausgehen können das jeder ein entsprechendes Office Programm zur Verfügung hat, finde ich es nicht schlecht die Textvariante zu behalten. So kann direkt gedruckt bzw. mit jedem Editor geöffnet werden. Zusätzlich kommt noch hinzu, dass die Veränderung der Header für die von @printpagestopdf eigentlich eingeführte Funktionalität nicht erforderlich ist und damit den Merge nur hinauszögert.

Ich wäre daher dafür das zu verlagern und habe dafür schon mal einen Thread angelegt, wenn ihr anderer Meinung seid, gerne den Thread schließen. https://github.com/wielebenwir/commonsbooking/discussions/1264

Reply:
Ja, vollständig richtig, der CSV download hat mit dem PR nichts zu tun. Die Änderung des vorgeschlagenen Filenamens war unnötige Piefigkeit bzgl. der Sprache. Nehme ich auf jeden Fall raus.
Ansonsten finde ich diskussion in separatem Thread gut.